API_Documentation.md

```diff
@@ -0,0 +1,533 @@
+
+# Documentation for GFW Layers & Datasets in API
```

Comment:
Can you put this in the docs folder? This when it has its own readable endpoint on github, and we can make a better layout.

Reply:
done